api/desecapi/management/commands/check-slaves.py

```diff
@@ -63,11 +66,17 @@ def handle(self, *args, **options):
         outdated_slaves = set()
 
         output = []
+        timeouts = {}
         for zone, local_serial in serials.items():
             outdated_serials = self.find_outdated_servers(zone, local_serial)
-            outdated_slaves.update(outdated_serials.keys())
+            for server, serial in outdated_serials.items():
+                if not serial and serial is not None:  # serial is False (error condition, instead of "not found"),
```

Comment:
how about "if serial is False"?

Reply:
I thought so too, but then I found https://legacy.python.org/dev/peps/pep-0008/#programming-recommendations:

> Don't compare boolean values to True or False using ==.
> 
> ```
> Yes:   if greeting:
> No:    if greeting == True:
> Worse: if greeting is True:
> ```

mh.

Reply:
The rule does not apply, as it is not (strictly) a Boolean value. No?

Reply:
fair enough, changed according to your suggestion